package.json

```diff
@@ -57,7 +57,7 @@
     "husky": "^7.0.4",
     "jest": "^27.5.1",
     "lint-staged": "^12.3.4",
-    "node-sass": "7.0.3",
+    "node-sass": "8.0.0",
```

Comment:
Since this is a development dependencies, we can leave this the lowest supported version

Reply:
Agreed